examples/nine-pt-stencil-solver/doc/intro.dox

```diff
@@ -27,7 +27,7 @@ On any node, where neighbor is on the border, the neighbor is replaced with a
 example a node with a neighborhood of only edge nodes may look like this
```

Comment:
Unfortunately, I am not allowed to comment on the line 13 of this file, so this line must do.

The Math block you have here should be changed to something like this:
```
$
    \Omega = (0,1)^2 \\
    \Omega_b = [0,1]^2 \text{  (with boundary)} \\
    \partial\Omega = \Omega_b \backslash \Omega \\
    u : \Omega_b -> R \\
    u'' = f \in \Omega \\
    u = u_D \text { on } \partial\Omega
$
```
That way, you have the proper newlines (with `\\`), have the text as text (with `\text{}`) and not as part of the fomular, have some spacing (thanks to the spaces inside `\text{}`), have the `in` as the appropriate symbol (with `\in`).
I am not sure if you also wanted to have the `on` as some kind of symbol, which is why I left it as a text.

Reply:
Isn't it double $ for multiple line equations? `$$`

https://www.overleaf.com/learn/latex/Mathematical_expressions?nocdn=true

Reply:
I was just looking at the 3pt_example doc, and there, we also use a single `$`.
https://github.com/ginkgo-project/ginkgo/blob/develop/examples/three-pt-stencil-solver/doc/intro.dox#L5

Reply:
Ah I missed the text on in my pushes. Technically it's still in that domain, but I saw this choice of words for domains and its boundaries in some of the finite element books. It's not wrong to say that it's in that domain.

Reply:
The in was also meant as text. But as I said, it really doesn't matter. So I'll change both.